PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutClient.java

```diff
@@ -1,14 +1,460 @@
+     *
+     * @param activity      Android FragmentActivity
+     * @param payPalRequest a {@link PayPalNativeRequest} used to customize the request.
+     */
+    public void tokenizePayPalAccount(@NonNull final FragmentActivity activity, @NonNull final PayPalNativeRequest payPalRequest) {
+        tokenizePayPalAccount(activity, payPalRequest, new PayPalNativeCheckoutFlowStartedCallback() {
+            @Override
+            public void onResult(@Nullable Exception error) {
+                if (error != null) {
+                    listener.onPayPalFailure(error);
+                }
+            }
+        });
+    }
+
+    /**
+     * Tokenize a PayPal account for vault or checkout.
+     * <p>
+     * Deprecated. Use {@link PayPalNativeCheckoutClient#tokenizePayPalAccount(FragmentActivity, PayPalNativeRequest)}
+     *
+     * @param activity      Android FragmentActivity
+     * @param payPalRequest a {@link PayPalNativeRequest} used to customize the request.
+     * @param callback      {@link PayPalNativeCheckoutFlowStartedCallback}
+     */
+    @Deprecated
+    public void tokenizePayPalAccount(@NonNull final FragmentActivity activity, @NonNull final PayPalNativeRequest payPalRequest, @NonNull final PayPalNativeCheckoutFlowStartedCallback callback) {
+        if (payPalRequest instanceof PayPalNativeCheckoutRequest) {
+            sendCheckoutRequest(activity, (PayPalNativeCheckoutRequest) payPalRequest, callback);
+        } else if (payPalRequest instanceof PayPalNativeCheckoutVaultRequest) {
+            sendVaultRequest(activity, (PayPalNativeCheckoutVaultRequest) payPalRequest, callback);
+        }
+    }
+
+    /**
+     * @param activity              Android FragmentActivity
+     * @param payPalCheckoutRequest a {@link PayPalNativeCheckoutRequest} used to customize the request.
+     * @param callback              {@link PayPalNativeCheckoutFlowStartedCallback}
+     * @deprecated Use {@link PayPalNativeCheckoutClient#tokenizePayPalAccount(FragmentActivity, PayPalNativeRequest, PayPalNativeCheckoutFlowStartedCallback)} instead.
+     * Starts the One-Time Payment (Checkout) flow for PayPal.
+     */
+    @Deprecated
+    public void requestOneTimePayment(@NonNull final FragmentActivity activity, @NonNull final PayPalNativeCheckoutRequest payPalCheckoutRequest, @NonNull final PayPalNativeCheckoutFlowStartedCallback callback) {
+        tokenizePayPalAccount(activity, payPalCheckoutRequest, callback);
+    }
+
+    /**
+     * @param activity           Android FragmentActivity
+     * @param payPalVaultRequest a {@link PayPalNativeCheckoutVaultRequest} used to customize the request.
+     * @param callback           {@link PayPalNativeCheckoutFlowStartedCallback}
+     * @deprecated Use {@link PayPalNativeCheckoutClient#tokenizePayPalAccount(FragmentActivity, PayPalNativeRequest, PayPalNativeCheckoutFlowStartedCallback)} instead.
+     * Starts the Billing Agreement (Vault) flow for PayPal.
+     */
+    @Deprecated
```

Comment:
I'm assuming these deprecated methods are an android thing or can these all be removed?

Reply:
This is a good callout. We should simplify the interface where we can. It is `next-major-version` material in the OG PayPal module but since this is new we should remove deprecated API.

Reply:
Removed the deprecated methods